search-ui/tests/unit/Breadcrumb.spec.ts

```diff
@@ -0,0 +1,50 @@
+// Libraries
+import { RouteNames } from '@/enums'
+import { mount } from '@vue/test-utils'
+import { BreadcrumbIF } from '@bcrs-shared-components/interfaces'
+import { BcrsBreadcrumb } from '@/bcrs-common-components'
+import { SearchDashboardBreadcrumb, SearchSIGNINBreadcrumb } from '@/bcrs-common-components/resources'
+import store from '@/store'
+
+
+describe('Breadcrumb.vue', () => {
+
+  let wrapper;
+  const breadcrumbs : BreadcrumbIF[] = [
+    {
+      text: SearchSIGNINBreadcrumb.text,
+      to: { name: RouteNames.SIGN_IN }
+    },
+    {
+      text: SearchDashboardBreadcrumb.text,
+      to: { name: RouteNames.DASHBOARD }
+    }
+  ]
+
+  wrapper = mount(BcrsBreadcrumb, {
+    props: { breadcrumbs },
+    global: {
+      provide: {
+        store: store
+      },
+      mocks: {
+        $route: {
+          name: RouteNames.DASHBOARD
+        }
+      }
+    },
+    shallow: true  // stubs out children components
+  })
+
+  it('present page is dashboard so should display breadcrumb', () => {
+    expect(wrapper.exists()).toBe(true)
+  })
+
+  it('dashboard back should be redirect to sign in', () => {
+    expect(wrapper.vm.backUrl()).toBe(RouteNames.SIGN_IN)
+  })
+
+  it('present page is dashboard so should in active this tab', () => {
+    expect(wrapper.vm.isActiveCrumb({ text: SearchDashboardBreadcrumb.text , to: { name: RouteNames.DASHBOARD } })).toBe(false)
```

Comment:
Can you verify the route in this test first please?

Reply:
I have changed method and its now isLast, giving me the last child so even in this method, I cannot verify route. I can verify index